lib/sli/rcsinfo.sli

```diff
@@ -0,0 +1 @@
+statusdict /rcsinfo (no_rcsinfo_available) put
```

Comment:
This is an empty file needed by the version tracking system which gets updated somewhere through the make files. On other people's system, it appears to end up empty --- and on mine, it ends up filled. I'd make a separate issue of it (@jougs).

So in short, it is in git, so it does get committed --- I can splice the update out, but I'm not sure what the point of this file is vis-a-vis github now.